src/components/Fruit.js

```diff
@@ -0,0 +1,40 @@
+import './Fruit.css';
+
+function Fruit({image, title, fruitAmount, fruitSetter}) {
+    const increase = () => {
+        fruitSetter(fruitAmount => fruitAmount + 1)
+    }
+
+    const decrease = () => {
+        fruitAmount > 0? fruitSetter(fruitAmount => fruitAmount -1): fruitSetter(0)
```

Comment:
Als `fruitAmount` 0 is, dan hoef je sowieso geen functie uit te voeren. Je zou dus zoiets kunnen doen:
```
if (fruitAmount > 0){
    fruitSetter(curr => curr - 1);
}
```